<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formDemoIpw</class>
 <widget class="QWidget" name="formDemoIpw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Plot Demo</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="text">
         <string>open file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkGrid">
        <property name="text">
         <string>show grid</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>496</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
